frontend/src/shared/baseUrl.js

```diff
@@ -0,0 +1 @@
+export const baseUrl = 'http://localhost:3001/';
```

Comment:
Please try to make this constant to a constant file where we store all the application constants.